### ISSUE TITLE
Shutting down keras_learning_phase in Resnet50

### ISSUE THREAD

**Issue by ghost** [fchollet/deep-learning-models#79]
I found a point in the ResNet50 in Keras. When you try to fine-tune the ResNet with your data, the first block of batch normalization has strange outputs!!! The network try to share the output of first batch normalization layer with other batch normalization layers in the network. If you check the Graph in Tensorboard, It is clear that there is a problem with this network. But I could not understand if the problem comes from tensorboard ot resnet itself!!!

**Comment by ghost** [comment]
Is there any way to shut down these sharing weights in batch normalization layers?! I mean how can I shut down the keras_learning_phase ("DT_BOOL"}) witch switches the batch normalization layers together
